src/main/target/SPRACINGF3/target.h

```diff
@@ -54,6 +54,9 @@
 #define USE_MAG_AK8975
 #define USE_MAG_HMC5883
 #define MAG_HMC5883_ALIGN CW270_DEG
+// test for ist8310 , ucomment it
```

Comment:
I use the HMC5883 and IST8310 in the same board, and I think I shouldn't change the default setting of SPRACINGF3.
therefore, I comment the ist8310 part.
Do I uncomment?